apps/web/src/services/dashboards/dashboard-detail/DashboardDetailPage.vue

```diff
@@ -167,6 +171,24 @@ const getDashboardData = async (dashboardId: string, force = false) => {
         await SpaceRouter.router.push({ name: DASHBOARDS_ROUTE.ALL._NAME });
     }
 };
+const updateDashboardLabel = async (labels: string[]) => {
+    try {
+        const isProjectDashboard = props.dashboardId?.startsWith('project');
+        if (isProjectDashboard) {
+            await SpaceConnector.clientV2.dashboard.projectDashboard.update({
+                project_dashboard_id: props.dashboardId,
+                labels,
+            });
+        } else {
+            await SpaceConnector.clientV2.dashboard.domainDashboard.update({
+                domain_dashboard_id: props.dashboardId,
+                labels,
+            });
+        }
+    } catch (e) {
+        ErrorHandler.handleError(e);
+    }
+};
```

Comment:
Question: is there any reason to separate `updateDashboardLabel` and `handleUpdateLabels`?

Reply:
I just tried to separate event func and api func... but it doesn't matter if they are combined.